src/SprykerEco/Yves/Computop/Form/DataProvider/PayPalExpressFormDataProvider.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Form\DataProvider;
+
+use Generated\Shared\Transfer\CustomerTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+
+class PayPalExpressFormDataProvider extends AbstractFormDataProvider
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function getData(AbstractTransfer $quoteTransfer): QuoteTransfer
+    {
+        if ($quoteTransfer->getPayment() === null) {
+            $paymentTransfer = new PaymentTransfer();
+            $quoteTransfer->setPayment($paymentTransfer);
+        }
+
+        if (!$this->isValidPayment($quoteTransfer)) {
+            $paymentTransfer = $quoteTransfer->getPayment();
+            /** @var \Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer $computopTransfer */
+            $computopTransfer = $this->mapper->createComputopPaymentTransfer($quoteTransfer);
+            $paymentTransfer->setComputopPayPalExpress($computopTransfer);
+            $quoteTransfer->setPayment($paymentTransfer);
+            $quoteTransfer->setCustomer(new CustomerTransfer());
+            $this->quoteClient->setQuote($quoteTransfer);
+        }
+
+        return $quoteTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Spryker\Shared\Kernel\Transfer\TransferInterface
+     */
+    protected function getComputopPayment(QuoteTransfer $quoteTransfer)
```

Comment:
please do not introduce one line methods like this
this wrapper makes no sense

Reply:
This method comes from abstract parent class `AbstractFormDataProvider`